src/main/java/org/example/models/JobRoleResponse.java

```diff
@@ -0,0 +1,72 @@
+package org.example.models;
+
+import java.util.Date;
+
+public class JobRoleResponse {
+    private int roleId;
+    private String roleName;
+    private String locations;
+    private String capability;
+    private String band;
+    private Date closingDate;
+
+    public JobRoleResponse(final int roleId, final String roleName,
+                   final String locations,
+                   final String capability,
+                   final String band, final Date closingDate) {
```

Comment:
Shouldn't this be an enum as it's an enum in the database

Reply:
No its returned as a string